examples/Training/java/src/training/ReadData.java

```diff
@@ -265,6 +271,33 @@ private void loadPlate() throws Exception {
         }
     }
 
+    /**
+     * Load the MapAnnotations of the currently logged in user.
+     */
+    private void loadMapAnnotations(long imageId) throws Exception {
+        BrowseFacility browse = gateway.getFacility(BrowseFacility.class);
+        ImageData image = browse.getImage(ctx, imageId);
+
+        // load only this user's annotations
+        List<Long> userIds = new ArrayList<Long>();
+        userIds.add(this.user.getId());
+
+        // load only MapAnnotations
+        List<Class<? extends AnnotationData>> types = new ArrayList<Class<? extends AnnotationData>>();
+        types.add(MapAnnotationData.class);
+
+        MetadataFacility metadata = gateway.getFacility(MetadataFacility.class);
+        List<AnnotationData> annotations = metadata.getAnnotations(ctx, image,
+                types, userIds);
+        for (AnnotationData annotation : annotations) {
+            MapAnnotationData mapAnnotation = (MapAnnotationData) annotation;
+            List<NamedValue> list = (List<NamedValue>) mapAnnotation
+                    .getContent();
+            for (NamedValue namedValue : list)
+                System.out.println(namedValue.name + ": " + namedValue.value);
+        }
```

Comment:
Small suggestion: maybe here within the outer loop can add a `System.out.println()` to put a blank line in between different annotations so we know which keys and values came from the same? Certainly not a blocker, just a thought.